### DISCUSSION TITLE
Upgrade to 3 "Must use import to load ES Module:"

### DISCUSSION THREAD

**Discussion by kaihendry** [serverless/serverless#10696] [General]
I upgraded to serverless 3.2 from 2.65 and now I see all these "Must use import to load ES Module:" errors I didn't see before.

Nothing in https://www.serverless.com/framework/docs/guides/upgrading-v3 about it.

What's going on?

**Comment by kaihendry** [comment]
Sorry it was node-fetch to blame https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md

**Reply by neverendingqs** [reply]
Double-check: it was `node-fetch` in your project, or `node-fetch` that `serverless@3.2.0` pulls in?

**Reply by kaihendry** [reply]
node-fetch in my project, sorry
